server/tabular.js

```diff
@@ -51,7 +51,7 @@ Meteor.publish("tabular_genericPub", function (tableName, ids, fields) {
   return table.collection.find({_id: {$in: ids}}, {fields: fields});
 });
 
-Meteor.publish("tabular_getInfo", function(tableName, selector, sort, skip, limit) {
+Meteor.publish("tabular_getInfo", function(tableName, selector, sort, skip, limit, throttleRefresh) {
```

Comment:
Just look up `throttleRefresh` from the `Tabular.Table` object (`table` variable below)